### PR TITLE
Modified the aliquot element so that it checks for enough initial sol…

### LINKED ISSUE #186: Aliquot element should check that there is enough solution to use in the case of Replica Plates being made
At the moment, the Aliquot element will check to ensure that there is enough solution for the number and volume specified for the aliquots. However, it only does the check assuming that there is one plate being made. If the user specifies more than one Replica Plate, the solution may run out without an error being thrown.

### PR DESCRIPTION
**Pull request by josi-asae**
…ution for all aliquots made over replicate plates. Also fixed a few typos and ran anthabuild which modified some bundle files Closes #186

### CODE REVIEW

File: starter/Aliquot/Aliquot.an [hunk 1]
```diff
@@ -65,9 +65,16 @@ Setup {
...
+	residualVol := SolutionVolume.SIValue() * 0.10
+
+	// Here we're doing some maths to work out what the possible number of aliquots is that we can make given the volume specified and the volume of solution we have.
+	// We round this number down to the nearest number of aliquots.
+	number := (SolutionVolume.SIValue() - residualVol) / VolumePerAliquot.SIValue()
 	possiblenumberofAliquots, _ := wutil.RoundDown(number)
-	if possiblenumberofAliquots < NumberofAliquots {
+	// The total number of aliquots to be made is the number specified by the user for each of the Replica Plates being made.
+	if possiblenumberofAliquots < (NumberofAliquots * NumberOfReplicaPlates) {
 		Errorf("Not enough solution for this many aliquots")
```
Comment (jarpinosynthace) [inline starter/Aliquot/Aliquot.an]: I think we should update the error message to say something along the lines of "Not enough solution for this many aliquots, you have specified X ul but require Y ul due to input plate residual volume"

**Comment by jarpinosynthace** [comment]
@synthacebot test this please

File: starter/Aliquot/Aliquot.an [hunk 2]
```diff
@@ -65,10 +65,20 @@ Setup {
 // for every input
 Steps {
 
-	number := SolutionVolume.SIValue() / VolumePerAliquot.SIValue()
+	// We need to make sure that we have enough solution after subtracting the residual volume of solution left in the input plate.
+	// In future this will be calculated explicitly, but here we are estimating it as 10% extra for simplicity.
```
Comment (chr1sgrant) [inline starter/Aliquot/Aliquot.an]: 10% could be wildly wrong in some cases; for example the trough has a residual volume of 5ml with the Gilson.

Comment (jarpinosynthace) [inline starter/Aliquot/Aliquot.an]: Yea this was my concern...can you think of a better way to represent it or solve this problem?

Comment (josi-asae) [inline starter/Aliquot/Aliquot.an]: @jarpinosynthace and I were chatting about this. The issue is that you need to know what kind of input plate you have in order to calculate residual volume. However, input plates are specified by the system unless the user asserts something in the config, which elements don't have access to (unless I am mistaken, in which case I'd love to learn how).

**Pull request closed without merging**